docs/data/toolpad/core/introduction/overview.md

```diff
@@ -4,7 +4,7 @@ title: Toolpad Overview
 
 # Toolpad Core
 
-<p class="description">A framework to build full-stack apps, fast. It can be used as a Next.js starter, or integrated within an existing React project to build dashboards and internal tools. </p>
```

Comment:
@prakhargupta1 This title is reported as too long. We need a shorter, more descriptive one.

Reply:
Changed it to: A set of full-stack components to build react dashboards.